smk-edit/static/components/edit-item-details-layer-esri-dynamic.js

```diff
@@ -4,6 +4,47 @@ export default importComponents( [
 ] ).then( function () {
     return vueComponent( import.meta.url, {
         props: [ 'itemId' ],
+        computed: {
+            hasLabels: function() {
+                const layer = this.$store.getters.configLayer( this.itemId );
+                if (layer.dynamicLayers) {
+                    for (const dynamicLayer of layer.dynamicLayers) {
+                        let dynamicJson = JSON.parse(dynamicLayer);
+                        if(dynamicJson.drawingInfo.labelingInfo) {
+                            return true;
+                        }
+                    };
+                }
+                return false;
+            },
+            showLabels: {
+                get: function () {
+                    const layer = this.$store.getters.configLayer( this.itemId );
+                    if (layer.dynamicLayers) {
+                        for (const dynamicLayer of layer.dynamicLayers) {
+                            const dynamicJson = JSON.parse(dynamicLayer);
+                            if(dynamicJson.drawingInfo.labelingInfo && dynamicJson.drawingInfo.showLabels) {
```

Comment:
Same issue here, if `dynamicInfo.drawingInfo` is `undefined` you will run into an exception here.